.github/workflows/dev-source-web-tool-scripts.yaml

```diff
@@ -0,0 +1,48 @@
+name: Dev source web-tool scripts
```

Comment:
@cjyetman, FYI this workflow is now in a different file. The one we've used so far remains intact.

Reply:
thanks. don't kill me, but... I'd prefer two separate PRs, one with the addition of a new workflow file, and one with all of the changes to the existing workflow file. I think that makes it easier to separate them contextually, and would be easier to understand how and why changes were made.

Reply:
actually, forget that... I see that is the case now... must've had an old diff cached in my browser or something and it looked like both were included here